vaxrank/vaccine_peptide.py

```diff
@@ -33,24 +35,34 @@ class VaccinePeptide(VaccinePeptideBase):
     """
     VaccinePeptide combines the sequence information of MutantProteinFragment
     with MHC binding predictions for subsequences of the protein fragment.
+
+    The resulting lists of mutant and wildtype epitope predictions are sorted by ic50.
     """
     def __new__(
             cls,
             mutant_protein_fragment,
             epitope_predictions,
+            num_mutant_epitopes_to_keep=10000,
             min_epitope_score=0):
+        # only keep the top k epitopes
+        mutant_epitope_predictions = sorted([
+            p for p in epitope_predictions if p.overlaps_mutation and not p.occurs_in_reference
+        ], key=attrgetter('ic50'))[:num_mutant_epitopes_to_keep]
```

Comment:
Note for the future: we'll at some point get a request to change this to sort by percentile rank instead.

Reply:
Change this or add an option to sort by percentile rank?